fivox/cuda/simpleLFP.cu

```diff
@@ -104,7 +130,8 @@ float simpleLFP( const float* posX, const float* posY, const float* posZ,
                           volInfo.dimensions.z;
 
     // 1D grid of 1D blocks
-    dim3 blockSize( 512, 1 );
+    unsigned int nBlocks = 512;
+    dim3 blockSize( nBlocks, 1 );
```

Comment:
I kept the form of two dimensions here because I thought it would be the right way to do it in the end, but apparently it doesn't bring any performance (even when using a value different than 1 for 'y').
I think the code is simpler if we just use unsigned ints directly for the kernel dimensions.